.circleci/config.yml

```diff
@@ -82,6 +190,14 @@ workflows:
           filters:
             tags:
               only: /.*/
+      # - build-darwin-go-1.12:
```

Comment:
why is this disabled?

Reply:
because on this linux VM in the CI cannot cross compile the darwin build because of CGO in Ethash. There is potentially an inconsistent and tedious workaround to use a mac VM through CircleCI and configure go but I would argue against it because Ethash should be migrated over and the build for linux and darwin are basically identical and tested locally.

The windows and mac build will be able to be uncommented and functional once Ethash is migrated from c to go